.github/workflows/release.yml

```diff
@@ -16,6 +16,16 @@ jobs:
         run: |
           bundle install
           bundle exec rake
+      - name: Increase version number
+        run: |
+          VERSION=$(git tag --list --sort=version:refname 'v*' | tail -1)
+          sed -i "5s/.*/    VERSION = '$VERSION'/" lib/asciidoctor/chart/version.rb
+      - name: Commit & Push updated version.rb
+        run: |
+          git add .
+          git commit -m "Updated version in version.rb"
```

Comment:
I think we should release on a `workflow_dispatch`:

https://github.com/asciidoctor/asciidoctor/blob/4bab183b9f3fad538f86071b2106f3b8185d3832/.github/workflows/release.yml#L2-L7


That way we can commit and tag. Otherwise we will commit the version after the tag which is not great.